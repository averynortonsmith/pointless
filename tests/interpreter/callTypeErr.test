
output = 123(3) |> println
