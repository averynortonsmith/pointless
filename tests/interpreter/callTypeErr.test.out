-------------------------------------------------------------------------------
Type Error:

Expected type 'PtlsBuiltIn or PtlsFunc', got 'PtlsNumber'
-------------------------------------------------------------------------------

At (line 2 column 10) in 'tests/interpreter/callTypeErr.test'
output = 123(3) |> println
         ^
At (line 2 column 13) in 'tests/interpreter/callTypeErr.test'
output = 123(3) |> println
            ^
